ui/src/components/core/core-modal.ts

```diff
@@ -109,9 +118,14 @@ export class CoreModal extends LitElement {
     dialog > div {
       padding: 24px;
     }
+
+    :host([hasNoPadding]) > dialog > div {
+      padding: 0;
+    }
```

Comment:
This could be shortened - not essential, but is IMO easier to read and reason about this way.
```suggestion
    :host(:not([hasNoPadding])) dialog > div {
      padding: 24px;
    }
```